tests/FakeItEasy.Tests/Creation/DummyValueResolverTests.cs

```diff
@@ -229,7 +229,6 @@ public void Should_favor_the_widest_constructor_when_activating()
         }
 
         [Theory]
-        [InlineData(typeof(void))]
         [InlineData(typeof(Func<int>))]
         [InlineData(typeof(Action))]
         public void Should_return_failed_result_for_restricted_types(Type restrictedType)

```

Comment:
Not sure what this test is doing. Why would `Func<int>` or `Action` be "restricted" types? And what does it even mean?
(I can actually make a dummy `Action` without any issue, it just returns a fake `Action`.)
I think it's because the fake `IFakeObjectCreator` is not configured to handle those types, but the term "restricted" is confusing...

Reply:
I agree! Thankfully, I'm not directly to blame

```
 3 years ago    Patrik Hägne    c03786ca6│ 186│         [TestCase(typeof(void))]
                                         │ 187│         [TestCase(typeof(Func<int>))]
                                         │ 188│         [TestCase(typeof(Action))]
                                         │ 189│         public void Should_return_false_for_restricted_types(Type restrictedType)
                                         │ 190│         {
                                         │ 191│             // Arrange
                                         │ 192│
                                         │ 193│             // Act
                                         │ 194│             object dummy;
                                         │ 195│             var result = this.session.TryResolveDummyValue(restrictedType, out dummy);
                                         │ 196│
                                         │ 197│             // Assert
                                         │ 198│             Assert.That(result, Is.False);
                                         │ 199│         }
```

Reply:
It's possible that this test has outlived its usefulness, superseded by our many lovely specs.

Reply:
> 3 years ago    Patrik Hägne    c03786ca6

More like 13 years ago...



> It's possible that this test has outlived its usefulness, superseded by our many lovely specs.

Yes, I think we should remove it. I'll add a commit.

Reply:
oops. Missed the first character when gutting from powershell